shell/platform/tizen/BUILD.gn

```diff
@@ -255,24 +255,13 @@ embedder("flutter_tizen_common") {
   defines = [ "COMMON_PROFILE" ]
 }
 
-embedder("flutter_desktop_source") {
+embedder("flutter_tizen_source") {
   target_type = "source_set"
   use_evas_gl_renderer = true
```

Comment:
I think ecore_wl2 renderer should be preferred if we start to support `flutter_tizen_unittests` on all profile.
or we have to choose a renderer for each profile.

Reply:
Okay, I'll check if it works.

Reply:
The default value can be changed but doing so will break the existing CI unit tests. Let's apply this later in a separate PR.

Reply:
Thanks for checking it out.